lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ServiceRegistry.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import uk.gov.di.ipv.cri.common.library.persistence.DataStore;
+import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
+import uk.gov.di.ipv.cri.common.library.persistence.item.SessionItem;
+import uk.gov.di.ipv.cri.common.library.service.AuditEventFactory;
+import uk.gov.di.ipv.cri.common.library.service.AuditService;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.common.library.service.PersonIdentityService;
+import uk.gov.di.ipv.cri.common.library.service.SessionService;
+import uk.gov.di.ipv.cri.common.library.util.ListUtil;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KBVGatewayFactory;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KeyStoreLoader;
+
+import java.time.Clock;
+
+public class ServiceRegistry {
+    private final ConfigurationService configurationService;
+    private AuditService auditService;
+    private SessionService sessionService;
+    private Clock clock = Clock.systemUTC();
+    private boolean isLoaded;
+
+    public ServiceRegistry(ConfigurationService configurationService) {
+        this.configurationService = configurationService;
+    }
+
+    public KBVService getKBVService() {
+        return new KBVService(new KBVGatewayFactory().create(this.configurationService));
+    }
+
+    public KBVStorageService getKBVStorageService() {
+        return new KBVStorageService(this.configurationService);
+    }
+
+    boolean isLoaded() {
+        return isLoaded;
```

Comment:
I think the keystore loading logic can be done at init time rather than being deferred. 
Then the isLoaded / setLoaded logic can be removed